libsolutil/Whiskers.cpp

```diff
@@ -74,6 +75,17 @@ string Whiskers::render() const
 	return replace(m_template, m_parameters, m_conditions, m_listParameters);
 }
 
+void Whiskers::checkTemplateValid() const
+{
+	regex validTemplate("<[?!\\/][a-zA-Z0-9_$-]+(?:[^a-zA-Z0-9_$>-]|$)");
```

Comment:
While it looks ugly, probably should use `paramRegex()` for identifier part, as that's what other regexes in this file do.

Maybe this needs a refactoring in a separate PR, let's keep it as-is but leave a comment that it must match `paramRegex`.

Reply:
Ahh I see, totally forgot about `paramRegex`. I can refactor this in another PR